tests/cases/conformance/jsx/tsxSfcReturnArray.tsx

```diff
@@ -0,0 +1,17 @@
+// @filename: file.tsx
+// @jsx: preserve
+// @module: amd
+// @noLib: true
+// @skipLibCheck: true
+// @libFiles: react.d.ts,lib.d.ts
+
+import React = require('react');
+
+const Foo = (props: any) => [<span />, <span />];

```

Comment:
This would warn at runtime due to missing `key` on the elements, but does it matter? :)

Reply:
You're right, this would warn at runtime. I guess it is fine for the test here as it only checks compiling...?